src/model/OutputTableAnnual.hpp

```diff
@@ -0,0 +1,141 @@
+/***********************************************************************************************************************
+*  OpenStudio(R), Copyright (c) Alliance for Sustainable Energy, LLC.
+*  See also https://openstudio.net/license
+***********************************************************************************************************************/
+
+#ifndef MODEL_OUTPUTTABLEANNUAL_HPP
+#define MODEL_OUTPUTTABLEANNUAL_HPP
+
+#include <model/ModelAPI.hpp>
+#include "ModelObject.hpp"
+
+namespace openstudio {
+namespace model {
+
+  class Schedule;
+
+  namespace detail {
+
+    class OutputTableAnnual_Impl;
+
+  }  // namespace detail
+
+  class MODEL_API AnnualVariableGroup
+  {
+   public:
+    AnnualVariableGroup(std::string variableorMeterorEMSVariableorField, std::string aggregationType = "SumOrAverage", int digitsAfterDecimal = 2);
```

Comment:
Name is required, but aggregation type and digits have default values.

Reply:
"SumOrAverage" is accoridng to the I/O reference, the most common option.

Digits after decimal = 2 is the E+ IDD default

Reply:
https://bigladdersoftware.com/epx/docs/24-2/input-output-reference/output-table-annual.html#field-aggregation-type-for-variable-or-meter

> SumOrAverage is probably the most common choice for aggregation type.

Reply:
```ruby
factory = OpenStudio::IddFileAndFactoryWrapper.new("EnergyPlus".to_IddFileType)
puts factory.getObject("Output:Table:Annual").get
```

```
Output:Table:Annual,
       \extensible:3
  A1, \field Name
[...]
  N1; \field Digits After Decimal
       \type integer
       \minimum 0
       \maximum 10
       \default 2
```